LDMP/models/datasets_delegate.py

```diff
@@ -80,13 +80,13 @@ def manageEditing(self, index: QModelIndex):
             return
 
         # activate editor
-        item = model.data(index, Qt.ItemDataRole)
+        item = model.data(index, Qt.DisplayRole)
```

Comment:
I would prefer to separete Display and Item role... btw because, in this case, I used ItemDataRole just to create editor (that at the end is a kind of visualization). I accept this modification.

Reply:
Thanks @luipir, the sort proxy model couldn't work with `Qt.ItemDataRole` that why I changed it.